core/app/models/spree/store.rb

```diff
@@ -76,8 +76,8 @@ class Store < Spree::Base
 
     def self.current(url = nil)
       ActiveSupport::Deprecation.warn(<<-DEPRECATION, caller)
-        `Spree::Store.current` is deprecated and will be removed in Spree 5
-        Spree::Stores::FindCurrent.new(url: "http://example.com") instead
+        `Spree::Store.current` is deprecated and will be removed in Spree 5.0
+        Please use `Spree::Stores::FindCurrent.new(url: "https://example.com").execute` instead
```

Comment:
I've aligned this deprecation warning with the other deprecation warnings. It didn't seem like:
`Spree::Stores::FindCurrent.new(url: "http://example.com") instead` is a complete sentence